### PR TITLE
tor, torify: add pages

### PR DESCRIPTION
**Pull request by eshedruf**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by spageektti** [comment]
Hello @eshedruf, thanks for your contribution to TLDR Pages! Can you please fix the error with trailing whitespace and sign the CLA?

File: pages/linux/tor.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- Start Tor relay:
+
+`tor --relay`
+
+- View Tor configuration:
+
+`tor --config`
+
+- Check Tor status:
```
Comment (spageektti) [inline pages/linux/tor.md:17]:
```suggestion
- Check Tor status:
```

**Comment by eshedruf** [comment]
> Hello @eshedruf, thanks for your contribution to TLDR Pages! Can you please fix the error with trailing whitespace and sign the CLA?

Ok, did it

File: pages/linux/tor.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# tor
+
+> Tor is free software for enabling anonymous communication.
```
Comment (spageektti) [inline pages/linux/tor.md:3]: I think the description should be changed. See [Style Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#example-descriptions).
```suggestion
> Enable anonymous communication through the Tor network.
```

File: pages/linux/tor.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+- Run as relay:
+
+`tor --relay`
+
+- Run as bridge:
+
+`tor --bridge`
+
+- Run as hidden service:
```
Comment (kbdharun) [inline pages/linux/tor.md:33]:
```suggestion
- Connect to the Tor network:

`tor`

- Start a Tor relay:

`tor --relay`

- View Tor configuration:

`tor --config`

- Check Tor status:

`tor --status`

- Run as client only:

`tor --client`

- Run as relay:

`tor --relay`

- Run as bridge:

`tor --bridge`

- Run as a hidden service:
```

File: pages/linux/torify.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+- Route traffic via Tor:
+
+`torify {{command}}`
+
+- Toggle Tor in shell:
+
+`. torify {{on|off}}`
+
+- Spawn Tor-enabled shell:
```
Comment (kbdharun) [inline pages/linux/torify.md:13]:
```suggestion
- Spawn a Tor-enabled shell:
```

File: pages/linux/torify.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+`torify --shell`
+
+- Check Tor-enabled shell:
```
Comment (kbdharun) [inline pages/linux/torify.md:17]:
```suggestion
- Check for a Tor-enabled shell:
```

File: pages/linux/torify.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+`torify show`
+
+- Specify Tor configuration file:
+
+`torify -c {{config-file}} {{command}}`
+
+- Use specific Tor SOCKS proxy:
```
Comment (kbdharun) [inline pages/linux/torify.md:25]:
```suggestion
- Use a specific Tor SOCKS proxy:
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some minor suggestions for this page.

**Comment by eshedruf** [comment]
@spageektti  @kbdharun your changes look good, I added them.

File: pages/linux/tor.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+# tor
+
+> Enable anonymous communication through the Tor network.
```
Comment (kbdharun) [inline pages/linux/tor.md:3]:
```suggestion
> Enable anonymous communication through the Tor network.
> More information: <https://manned.org/man/tor>.
```

File: pages/linux/torify.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
+# torify
+
+> Route network traffic through the Tor network.
```
Comment (kbdharun) [inline pages/linux/torify.md:3]:
```suggestion
> Route network traffic through the Tor network.
> More information: <https://manned.org/man/torify>.
```

File: pages/linux/torify.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+`torify -c {{config-file}} {{command}}`
+
+- Use a specific Tor SOCKS proxy:
+
+`torify -P {{proxy}} {{command}}`
+
+- Redirect output to a file:
+
+`torify {{command}} > output.txt`
```
Comment (kbdharun) [inline pages/linux/torify.md:31]:
```suggestion
`torify {{command}} > {{path/to/output}}`
```
IG we can convert this to a placeholder. Also, I have removed the extension as any extensions can be used.

**Review by kbdharun — COMMENTED** [review]
The pages almost LGTM, we could add more information links for additional information.

**Review by kbdharun — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**